pyproject.toml

```diff
@@ -10,27 +10,48 @@ force-exclude = '''
 [tool.codespell]
 ignore-words-list = 'astroid,crasher,asend'
 
-[tool.flake8]
-extend-ignore = ['D', 'E', 'W', 'F403', 'F405', 'F821', 'F822']
-per-file-ignores = [
-  'trio/__init__.py: F401',
-  'trio/_core/__init__.py: F401',
-  'trio/_core/_tests/test_multierror_scripts/*: F401',
-  'trio/abc.py: F401',
-  'trio/lowlevel.py: F401',
-  'trio/socket.py: F401',
-  'trio/testing/__init__.py: F401'
+[tool.ruff]
+target-version = "py38"
+respect-gitignore = true
```

Comment:
Note this is the default behavior

Reply:
I thought it would be better to have at least some of the defaults set manually for if said settings are ever changed in a future PR.